### PR TITLE
Guide: Svelte with Layer0

### PR DESCRIPTION
**Pull request by rishi-raj-jain**

### CODE REVIEW

File: guides/svelte.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+npm install
+```
+
+You can verify your app works by running it locally:
+
+```bash
+npm run dev
+```
+
+You should see an output like this in the terminal:
```
Comment (jeffpatzer) [inline guides/svelte.md:41]: @rishi-raj-jain what's the need for showing the output of running `npm run dev`?

File: guides/svelte.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+    error(error) {
+      console.error('Error during service worker registration:', error)
+    },
+  })
+}
+```
+
+and to include the service worker in the app, edit `main.js` (in the `src` folder) as follows:
+
+```diff
```
Comment (jeffpatzer) [inline guides/svelte.md:151]: diff?

File: guides/svelte.md (hunk 1 above)
Comment (rishi-raj-jain) [reply guides/svelte.md:41]: Ah it's inspired from https://docs.layer0.co/guides/vue#section_create_a_new_vue_js_app, but I guess here the output is too big.

File: guides/svelte.md (hunk 2 above)
Comment (rishi-raj-jain) [reply guides/svelte.md:151]: Used to highlight code differences in markdown: https://stackoverflow.com/questions/40883421/diff-syntax-highlighting-in-github-markdown?answertab=votes#tab-top

File: guides/svelte.md (hunk 1 above)
Comment (rishi-raj-jain) [reply guides/svelte.md]: removed.

**Review by tristanlee85 — APPROVED** [review]

File: guides/svelte.md (hunk 2 above)
Comment (jeffpatzer) [reply guides/svelte.md:119]: Cool feature, I missed the `+ +`

**Pull request merged**